.github/workflows/release-server-to-docker.yml

```diff
@@ -30,27 +39,36 @@ jobs:
         shell: bash
         id: get-docker-sha
-        run: echo "DOCKER_SHA=$(docker pull mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} > /dev/null 2>&1 && docker inspect --format='{{index .RepoDigests 0}}' mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} | cut -d '@' -f 2)" >> $GITHUB_ENV
-      - name: Sophos Factory pipeline
-        uses: sophos/factory-run-pipeline@v2
-        with:
-          project_id: 601c48ad71a5df94e698a9ad
-          job_id: 644185303992f67e1d9412f2
-          api_token: ${{ secrets.SOPHOS_FACTORY_TOKEN }}
-          variables: |
-            {
-              "gitlabHost": "repo1.dso.mil",
-              "heimdallVersion": "${{ steps.format-tag.outputs.replaced }}",
-              "heimdallDockerURL": "docker://docker.io/mitre/heimdall2@${{ env.DOCKER_SHA }}"
-            }
+        run: |
+          echo "DOCKER_SHA=$(docker pull mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} > /dev/null 2>&1 && docker inspect --format='{{index .RepoDigests 0}}' mitre/heimdall2:${{ steps.format-tag.outputs.replaced }} | cut -d '@' -f 2)" >> $GITHUB_ENV
+      - name: Make working directory for Iron Bank changes
+        run: mkdir ../ironbank_heimdall
+      - name: Clone and update local copy of Iron Bank repo to have the latest tags and digest
+        working-directory: ../ironbank_heimdall
+        run: |
+          git clone https://repo1.dso.mil/dsop/mitre/security-automation-framework/heimdall2.git .
+          git switch -c "${{ env.VERSION_HYPHENATED }}" # swap to sha for mainline
+          # We are required to update all Heimdall version references and the Docker SHA to the latest
+          yq e -i '.args.HEIMDALL_VERSION="${{ steps.format-tag.outputs.replaced }}" | .tags[0]="${{ steps.format-tag.outputs.replaced }}" | .labels."org.opencontainers.image.version"="${{ steps.format-tag.outputs.replaced }}" | .resources[1].tag="mitre/heimdall2:${{ steps.format-tag.outputs.replaced }}" | .resources[1].url="docker://docker.io/mitre/heimdall2@${{ env.DOCKER_SHA }}"' hardening_manifest.yaml
+          sed -i s/HEIMDALL_VERSION=\.\*/HEIMDALL_VERSION=${{ steps.format-tag.outputs.replaced }}/ Dockerfile
+          git diff
+          git add hardening_manifest.yaml Dockerfile
+          git -c "user.name=Automated Heimdall Release" -c "user.email=saf@groups.mitre.org" commit -s -m "Updating Heimdall to ${{ steps.format-tag.outputs.replaced }}"
+      - name: Create issue, branch, and merge request for Iron Bank repo
+        working-directory: ../ironbank_heimdall
+        run: |
+          ISSUE=$(curl -X POST --header "PRIVATE-TOKEN: ${{ secrets.SAF_IRONBANK_REPO1_PAT }}" --header "Content-Type: application/json" --data '{"title": "Update Heimdall to ${{ steps.format-tag.outputs.replaced }}", "labels": "Status::Review"}' "https://repo1.dso.mil/api/v4/projects/${{ env.IRONBANK_HEIMDALL_PROJECT_ID }}/issues" | jq ".iid")
```

Comment:
new workflow process

if the issue title is new (ex. update heimdall to v3 and has a description that specifies the tag), then make the mr and update the issue to include the generated mr id - probably will require the issue description to be structured data in some capacity so we can process it
if the issue title is the same (ex. update heimdall to latest commit) then update the description (specify the new commit), make a new mr and update the issue with the new id, and take the old id and use it to close the old mr

Reply:
For now I no longer think that this will be an issue since workflows only trigger on human action.